west2nix.toml

```diff
@@ -104,25 +113,24 @@ groups = ["hal"]
 hash = "sha256-H2C+3ASsC5O/mA+O5EJqfoJgx0BhSN0W3OhWNHVQxRU="
 
 [[manifest.projects]]
-name = "hal_atmel"
-url = "https://github.com/zephyrproject-rtos/hal_atmel"
-revision = "aad79bf530b69b72712d18873df4120ad052d921"
-path = "modules/hal/atmel"
+name = "hal_ambiq"
+url = "https://github.com/zephyrproject-rtos/hal_ambiq"
+revision = "94dd874cd726ba8185a301e78337c5c39685123f"
+path = "modules/hal/ambiq"
 groups = ["hal"]
 
 [manifest.projects.nix]
-hash = "sha256-9mJjeX9l7orJvYXjW46YROTPvm389KKF3s1XYj7g+VM="
+hash = "sha256-1YDU9LqO5XqMnUtLcYUS/VK0FnCe243QsO9683MpzB4="
 
 [[manifest.projects]]
-name = "hal_espressif"
```

Comment:
**suggestion (blocking):** you should reactivate the Espressif HAL – this issue was fixed with https://github.com/zephyrproject-rtos/zephyr/pull/72186

Reply:
The issue is not fixed as of [b5f5fa6](https://github.com/zephyrproject-rtos/hal_espressif/tree/b5f5fa6d08780c0077407150c6685d6c9d70262c). The repo still contains a dangling submodule `micro-ecc`, with no `.gitmodules` specification (try running `git submodule` in that repo).
It's difficult to parse that repo since it contains multiple disjunct git trees, but it seems the `zephyr` branch has had this problem at least since 2022.